frontend/src/components/courses-list/CoursesListItem.js

```diff
@@ -54,7 +54,7 @@ class CoursesListItem extends Component {
                 Average progress:
               </span>
               <span className={styles['stat-value']}>
-                {this.props.averageProgress*100}%
+                {(this.props.averageProgress*100).toFixed(2)}%
```

Comment:
Why would you want to loose the precision `56.999999999999%`, is so precise :wink: